webpack.config.js

```diff
@@ -7,6 +7,8 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const autoprefixer = require('autoprefixer');
 const pxtorem = require('postcss-pxtorem');
 
+const uglifyOpts = { compress: { warnings: false }, sourceMap: true };
```

Comment:
I think that moving the options 20 lines away from creating the UglifyJsPlugin makes the code harder to read.